web/app/view/edit/Devices.js

```diff
@@ -81,11 +81,14 @@ Ext.define('Traccar.view.edit.Devices', {
     viewConfig: {
         enableTextSelection: true,
         getRowClass: function (record) {
-            var status = record.get('status');
+            var status = record.get('status'), result = '';
```

Comment:
I think it's better to swap this around. I prefer longer assignment at the end so it's easier to read the code.